judgels-frontends/raphael/src/routes/contests/contests/single/clarifications/ContestClarificationsPage/ContestClarificationsPage.tsx

```diff
@@ -20,21 +20,37 @@ import { selectContest } from '../../../modules/contestSelectors';
 import * as contestClarificationActions from '../modules/contestClarificationActions';
 
 import './ContestClarificationsPage.css';
+import { ClarificationFilterWidget } from '../../../../../../components/ClarificationFilterWidget/ClarificationFilterWidget';
```

Comment:
Can we please tidy up the imports?

- external imports
- internal imports
- blank line
- CSS import

Reply:
got it, will update